app/controllers/movie_controller.rb

```diff
@@ -0,0 +1,40 @@
+class MovieController < ApplicationController
+  def top_forty
+    top_40 = []
+    page = 1
+    2.times do
```

Comment:
I've been doing some research into pagination, however, I know you wanted to share out that there may be a different end point to use, so lets discuss that first